packages/amazon-vpc-cni-plugins/Cargo.toml

```diff
@@ -8,6 +8,10 @@ build = "../build.rs"
 [lib]
 path = "../packages.rs"
 
+[package.metadata.build-package]
+# Use the Git submodule commit from aws/amazon-ecs-agent rather than an upstream release.
+releases-url = "https://github.com/aws/amazon-ecs-agent/commits/master/amazon-vpc-cni-plugins"
```

Comment:
> Should we point to the source package itself?

I tried to find a way to link to the commit ID for the submodule itself, and as far as I can tell that's not possible with the GitHub web UI. This links to the commit history for the submodule, which felt like the next best thing since one of the changes in that commit will show the new project commit ID.

I can drop if it that's too subtle, but people keep trying to update this package so I wanted a more clear statement on where the correct version is supposed to come from.